modules/core/src/GVL.ts

```diff
@@ -401,21 +401,33 @@ export class GVL extends Cloneable<GVL> implements VendorList {
    * functionality and methods of this class.
    */
   public getJson(): VendorList {
-
-    return JSON.parse(JSON.stringify({
-      gvlSpecificationVersion: this.gvlSpecificationVersion,
-      vendorListVersion: this.vendorListVersion,
-      tcfPolicyVersion: this.tcfPolicyVersion,
-      lastUpdated: this.lastUpdated,
-      purposes: this.purposes,
-      specialPurposes: this.specialPurposes,
-      features: this.features,
-      specialFeatures: this.specialFeatures,
-      stacks: this.stacks,
-      dataCategories: this.dataCategories,
-      vendors: this.fullVendorList,
-    }));
-
+    const {
+        gvlSpecificationVersion,
+        vendorListVersion,
+        tcfPolicyVersion,
+        lastUpdated,
+        purposes,
+        specialPurposes,
+        features,
+        specialFeatures,
+        stacks,
+        dataCategories,
+        fullVendorList
+    } = this;
+
+    return {
+        gvlSpecificationVersion,
+        vendorListVersion,
+        tcfPolicyVersion,
+        lastUpdated,
+        purposes,
+        specialPurposes,
+        features,
+        specialFeatures,
+        stacks,
+        dataCategories,
+        vendors: fullVendorList
+    };
```

Comment:
This is not a deep clone of `VendorList`, right? We will return references to objects like `vendors` and anyone outside of the library could update  it, maybe it's not good. 

The `JSON.parse(JSON.stringify())` is ugly and very slow particular for the very big object like GVL. Maybe we can develop an alternative solution creating custom deep clone method in this case as we know the GVL structure?